tests/queries/test_explain.py

```diff
@@ -159,9 +159,7 @@ def test_mysql_text_to_traditional(self):
         self.assertEqual(len(captured_queries), 1)
         self.assertIn("FORMAT=TRADITIONAL", captured_queries[0]["sql"])
 
-    @unittest.skipUnless(
-        connection.vendor == "mysql", "MariaDB and MySQL >= 8.0.18 specific."
-    )
+    @unittest.skipUnless(connection.vendor == "mysql", "MySQL specific")
```

Comment:
This test passes on mariadb, so I think we can remove the skip. Debatable whether we should fix in this PR, but we are removing the dynamic skip for this test elsewhere in the PR.